whitepaper.md

```diff
@@ -1,33 +1,36 @@
+We present TEN, a next-generation Ethereum Layer 2 rollup protocol that introduces data confidentiality, computational privacy, and resistance to [Maximal Extractable Value (MEV)](https://ethereum.org/en/developers/docs/mev/) by leveraging hardware-based [Trusted Execution Environments (TEEs)](https://en.wikipedia.org/wiki/Trusted_execution_environment).
 
-The design of TEN ensures that hardware manufacturers do not have to be trusted for the safety of the ledger. If one manufacturer turns malicious or there is a breach in the TEE technology, the protocol falls back to the behaviour of a public blockchain that preserves the ledger's integrity but makes the transactions public.
+TEN represents a significant advancement in the design of decentralised systems by restoring data access controls, a foundational feature of Web2 that Web3 largely abandoned in the pursuit of transparency. In early computing, users interacted with isolated desktop machines. The internet ushered in a new paradigm: shared data and remote computation, gated by fine-grained access control. Platforms like Facebook and eBay flourished under this model. However, as Web3 reimagined the internet with open-state, composable systems, it discarded the principle of controlled data visibility—limiting its ability to support sensitive, real-world use cases.
 
-The design also focuses on preserving privacy for the limited period when it matters most, which removes the need for a privacy technique that is robust against all adversaries in perpetuity.
+By integrating programmable encryption, TEEs, and an Ethereum-compatible execution environment, TEN reintroduces access control into a fully decentralised setting. This enables encrypted, autonomous, and composable smart contracts that preserve user and application privacy without sacrificing decentralisation or composability. The protocol supports private shared state, confidential transactions, and short withdrawal periods, while maintaining the simplicity and performance characteristics of Optimistic rollups.
 
-TEN sits in what we believe is a sweet spot between the existing [rollup-based L2 offerings](https://ethereum.org/en/developers/docs/scaling/layer-2-rollups/) Optimistic and ZK Rollups. The use of [confidential computing techniques](https://www.intel.co.uk/content/www/uk/en/security/confidential-computing.html) coupled with economic incentives allows TEN to retain the performance and programming model simplicity of Optimistic rollups, and on top of that attain confidentiality, short withdrawal periods, and address MEV.
+Importantly, TEN’s trust model does not require perpetual belief in any single hardware vendor. If a TEE is compromised or a manufacturer behaves maliciously, the system gracefully degrades into a transparent blockchain, preserving ledger integrity while forfeiting privacy—a fallback consistent with the ethos of credible neutrality.
```

Comment:
I'd say this needs to be rephrased. To specify the two promises: The data privacy and integrity. And then explain what happens